src/data/checkInput.js

```diff
@@ -82,9 +82,15 @@ export default function checkInput({
     if (schema.type === 'array') {
         // check for empty array
         if (argument.length === 0) {
-            throw new Error(
+            //    throw new Error(
+            //        `Empty array: [ ${parameter} ] argument to [ ${module}() ] contains zero elements.`
+            //    );
+            // display a warning in teh console
+            //if (verbose) {
+            console.log(
                 `Empty array: [ ${parameter} ] argument to [ ${module}() ] contains zero elements.`
             );
+            //}
```

Comment:
```suggestion
            console.log(
                `Empty array: [ ${parameter} ] argument to [ ${module}() ] contains zero elements.`
            );
```
I try to avoid leaving behind ghosts of old code/other ideas. We have the old code in github history. The `if (verbose)` bit might be worth keeping (uncommented), though. I'm not sure if that's defined here, though.

Reply:
Completely agree, thanks for the catch!